script/release

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Usage: script/release [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease | from-git]
+#
+# - Triggers a version bump to the provided explicit version,
+#   or to the derived new version given the "bump level".
+#   Leverages npm-version to the bumping and git-commit/tag.
+#   Requires git to be configured for GPG signing.
+
+set -euo pipefail
+IFS=$'\n\t'
+
+BUMP=${1:-patch}
+RELNOTES="$(git rev-parse --git-dir)/RELNOTES"
+
+./script/release-notes
+
+printf -v MESSAGE 'Bats %%s\n\n%s' "$(sed -E 's/^#+[ \t]*//' "$RELNOTES")"
+
+npm version --sign-git-tag --message "$MESSAGE" "$BUMP"
```

Comment:
Does this bump the tag with the `v` in front? I didn't think `npm version` typically did that. I guess it just parses the last semver tag, bumps only the numbers, and leaves any prefix alone?

Reply:
`npm version` bumps based on the version from package.json, not based on any git tags. It does _create_ (and sign) git tags, but doesn't use them as a basis for bumping.

And the prefix for the git tag is a configurable option: https://docs.npmjs.com/misc/config#tag-version-prefix

Reply:
Ohhh...didn't know about that option, thanks!